app/services/mixpanel_analytics.rb

```diff
@@ -0,0 +1,21 @@
+module MixpanelAnalytics
+  def requested_feed(owner:, repo:)
+    event = 'Fetched feed'
+    properties = { owner: owner, repo: repo }
+    track(event, properties)
+  end
+
+  private
+
+  def track(event, properties)
+    tracker.track(user_id, event, properties)
+  end
+
+  def tracker
+    @tracker ||= Mixpanel::Tracker.new(ENV['MIXPANEL_TOKEN'])
+  end
+
+  def user_id
+    request.remote_ip
```

Comment:
All this data is for fun since I'm not charging anything for this. So I can experiment a little.